src/modules/storage/LedgerStorage.ts

```diff
@@ -1355,16 +1355,34 @@ export class LedgerStorage extends Storages {
                 .catch((err) => {
                     reject(err);
                 });
-
-            const utxo_manager: UTXOManager = new UTXOManager(utxo_array);
-            const getSum: Amount[] = await utxo_manager.getSum(JSBI.add(height.value, JSBI.BigInt(1)));
             const total_txs = await this.getTxCount(height, address, conn);
-            const accountInfo: IAccountInformation = {
-                total_balance: JSBI.add(JSBI.add(getSum[0].value, getSum[1].value), getSum[2].value),
-                total_spendable: JSBI.BigInt(getSum[0]),
-                total_frozen: JSBI.BigInt(getSum[1]),
+            let accountInfo: IAccountInformation = {
+                total_balance: JSBI.BigInt(0),
+                total_spendable: JSBI.BigInt(0),
+                total_frozen: JSBI.BigInt(0),
+                total_reward: JSBI.BigInt(0),
                 tx_count: total_txs[0].tx_count ? total_txs[0].tx_count : 0,
             };
+            utxo_array.map((elem) => {
+                switch (elem.type) {
+                    case OutputType.Payment: {
+                        accountInfo.total_spendable = JSBI.add(accountInfo.total_spendable, elem.amount.value);
+                        break;
+                    }
+                    case OutputType.Freeze: {
+                        accountInfo.total_frozen = JSBI.add(accountInfo.total_frozen, elem.amount.value);
+                        break;
+                    }
+                    case OutputType.Coinbase: {
```

Comment:
Coinbase transactions have empty sender(Inputs).

Reply:
For `Coinbase` transactions the single input is `hashFull(height)`

Reply:
when there's no senders, we do not deal with senders array

Reply:
transaction at block height has no senders, so that only receivers array is traversed

Reply:
Yes, I checked it.
https://v7.bosagora.io/blocks_from?height=288&max_blocks=1